### PR TITLE
SSD Tweaks

### PR DESCRIPTION
**Pull request by alsoandanswer**
- tweak: "SSD no longer immediately puts you to sleep - only after 5 minutes will your character drop on the ground. Also, adds an overlay to indicate when someone's SSD."

Note that the overlay will disappear when someone ghosts, since, well. They aren't waiting for the client to return, they've ghosted.
dreamseeker_84e7fwfWLA

### CODE REVIEW

**Review by KingOfThePing — APPROVED** [review]
5 minutes may be a bit much but it's as good as anything, nice tweak

**Comment by Yakunan** [comment]
Nice! That's really handy! Thank you!

**Comment by alsoandanswer** [comment]
!review

File: code/modules/mob/living/carbon/human/login.dm [hunk 1]
```diff
@@ -3,6 +3,8 @@
 	update_hud()
 	if(species)
 		species.handle_login_special(src)
+	if(client)
+		overlays -= image('icons/effects/effects.dmi', icon_state = "zzz_glow")
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/login.dm:7]: Why isn't this using remove/add_overlay?

**Review by NonQueueingMatt — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/login.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/modules/mob/living/carbon/human/login.dm:7]: Woops. Direct port fron Paradise. I'll fix it.

**Comment by alsoandanswer** [comment]
note to self add flick animation of bubble popping

**Comment by alsoandanswer** [comment]
dreamseeker_lUkccC4cbv

New Jraphics.

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -741,7 +741,7 @@
 		return 0
 
 	//SSD check, if a logged player is awake put them back to sleep!
-	if(species.show_ssd && (!client && !vr_mob) && !teleop)
+	if(species.show_ssd && (!client && !vr_mob) && !teleop && ((world.realtime - disconnect_time) >= 5 MINUTES)) //only sleep after 5 minutes, should help those with intermittent internet connections
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/life.dm:744]: 30 seconds or 1 minute should be enough.

Comment (alsoandanswer) [reply code/modules/mob/living/carbon/human/life.dm:744]: 5 minutes is to play it safe

especially with BYOND ads and whatnot

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**